subsys/net/lib/download_client/src/http_download_client.c

```diff
@@ -281,6 +281,12 @@ static int header_parse(struct download_client *client)
 		LOG_DBG("File size = %d", client->file_size);
 	}
 
+	p = strstr(client->buf, "Connection: close");
+	if (p) {
+		LOG_WRN("Peer closed connection, will attempt to re-connect");
```

Comment:
If it's a typical condition I don't think we should use a "warning" level here? INF or even DBG should be fine.

Reply:
I think warning is best here, since it's not really ideal to have to disconnect/reconnect.
If possible, it should be avoided by increasing the fragment buffer size.

Reply:
Ok, I buy it.

Reply:
I could add one warning when the file size is obtained, if it is larger than 100 fragments.
It would be good to mention it in the doc and  in the Kconfig options help text as well.

Reply:
Added a note in the documentation, the warning text in the log would have been too long.